docs/modelview/src/tableview/main.qml

```diff
@@ -0,0 +1,48 @@
+// M1>>
+import QtQuick 2.12
+// <<M1
+import QtQuick.Window 2.0
```

Comment:
2.12
you can use x.12 for every import, where x is the module version.  (In Qt 6 we might make it even more consistent, i.e. maybe every module will have version 6.y)

Reply:
This is explained earlier (and it's great with the x.12 - you have my vote for a 6.12 approach). However, when discussing the general update of versions, we decided against changing all the import statements to the latest version and instead show that features are added over time.